packages/block-library/src/rss/index.php

```diff
@@ -79,8 +79,17 @@ function render_block_core_rss( $attributes ) {
 		$list_items .= "<li class='wp-block-rss__item'>{$title}{$date}{$author}{$excerpt}</li>";
 	}
 
-	$classes           = 'grid' === $attributes['blockLayout'] ? ' is-grid columns-' . $attributes['columns'] : '';
-	$list_items_markup = "<ul class='wp-block-rss{$classes}'>{$list_items}</ul>";
+	$class = 'wp-block-rss';
+
+	if ( 'grid' === $attributes['blockLayout'] ) {
+		$class .= ' is-grid columns-' . $attributes['columns'];
+	}
+
+	if ( isset( $attributes['className'] ) ) {
```

Comment:
I think we need to check for an empty className. Right now, if we write a custom className and then we delete the custom class name the class field on the dom gets an empty space.

Reply:
```suggestion
	if ( ! empty( $attributes['className'] ) ) {
```